app/src/main/java/net/gsantner/opoc/util/FileUtils.java

```diff
@@ -405,19 +405,21 @@ public static boolean isTextFile(File file) {
      * Analyze given textfile and retrieve multiple information from it
      * Information is written back to the {@link AtomicInteger} parameters
      */
-    public static void retrieveTextFileSummary(File file, AtomicInteger numCharacters, AtomicInteger numLines) {
+    public static void retrieveTextFileSummary(File file, AtomicInteger numCharacters, AtomicInteger numLines, AtomicInteger numWords) {
         BufferedReader br = null;
         try {
             br = new BufferedReader(new FileReader(file));
             String line;
             while ((line = br.readLine()) != null) {
                 numLines.getAndIncrement();
                 numCharacters.getAndSet(numCharacters.get() + line.length());
+                numWords.getAndSet(numWords.get() + line.split("\\s+").length);
```

Comment:
does this skip every new line?

Reply:
It did not. A more recent commit should fix this.